ush/radmon_verf_angle.sh

```diff
@@ -0,0 +1,251 @@
+
+# Command line arguments.
+RAD_AREA=${RAD_AREA:-glb}
+REGIONAL_RR=${REGIONAL_RR:-0}	# rapid refresh model flag
+rgnHH=${rgnHH:-}
+rgnTM=${rgnTM:-}
+
+export PDATE=${1:-${PDATE:?}}
+
+echo " REGIONAL_RR, rgnHH, rgnTM = $REGIONAL_RR, $rgnHH, $rgnTM"
+netcdf_boolean=".false."
+if [[ $RADMON_NETCDF -eq 1 ]]; then
+   netcdf_boolean=".true."
+fi  
+echo " RADMON_NETCDF, netcdf_boolean = ${RADMON_NETCDF}, $netcdf_boolean"
+
+which prep_step
+which startmsg
+
+# Directories
+FIXgdas=${FIXgdas:-$(pwd)}
+EXECradmon=${EXECradmon:-$(pwd)}
+TANKverf_rad=${TANKverf_rad:-$(pwd)}
+
+# File names
+export pgmout=${pgmout:-${jlogfile}}
+touch $pgmout
+
+# Other variables
+SATYPE=${SATYPE:-}
+VERBOSE=${VERBOSE:-NO}
+LITTLE_ENDIAN=${LITTLE_ENDIAN:-0}
+USE_ANL=${USE_ANL:-0}
+
+
+if [[ $USE_ANL -eq 1 ]]; then
+   gesanl="ges anl"
+else
+   gesanl="ges"
+fi
+
+err=0
+angle_exec=radmon_angle.x
+shared_scaninfo=${shared_scaninfo:-$FIXgdas/gdas_radmon_scaninfo.txt}
+scaninfo=scaninfo.txt
+
+#--------------------------------------------------------------------
+#   Copy extraction program and supporting files to working directory
+
+$NCP ${EXECradmon}/${angle_exec}  ./
+$NCP $shared_scaninfo  ./${scaninfo}
+
+if [[ ! -s ./${angle_exec} || ! -s ./${scaninfo} ]]; then
+   err=2
+else
+#--------------------------------------------------------------------
+#   Run program for given time
+
+   export pgm=${angle_exec}
+
+   iyy=$(echo $PDATE | cut -c1-4)
+   imm=$(echo $PDATE | cut -c5-6)
+   idd=$(echo $PDATE | cut -c7-8)
+   ihh=$(echo $PDATE | cut -c9-10)
+
+   ctr=0
+   fail=0
+   touch "./errfile"
+
+   for type in ${SATYPE}; do
+
+      if [[ ! -s ${type} ]]; then
+         echo "ZERO SIZED:  ${type}"
+         continue
+      fi
+
+      for dtype in ${gesanl}; do
+
+         echo "pgm    = $pgm"
+         echo "pgmout = $pgmout"
+         prep_step
+
+         ctr=$(expr $ctr + 1)
+
+         if [[ $dtype == "anl" ]]; then
+            data_file=${type}_anl.${PDATE}.ieee_d
+            ctl_file=${type}_anl.ctl
+            angl_ctl=angle.${ctl_file}
+         else
+            data_file=${type}.${PDATE}.ieee_d
+            ctl_file=${type}.ctl
+            angl_ctl=angle.${ctl_file}
+         fi
+
+         if [[ $REGIONAL_RR -eq 1 ]]; then
+            angl_file=${rgnHH}.${data_file}.${rgnTM}
+         fi
+
+
+         rm input
+
+         nchanl=-999
+cat << EOF > input
+ &INPUT
+  satname='${type}',
+  iyy=${iyy},
+  imm=${imm},
+  idd=${idd},
+  ihh=${ihh},
+  idhh=-720,
+  incr=${CYCLE_INTERVAL},
+  nchanl=${nchanl},
+  suffix='${RADMON_SUFFIX}',
+  gesanl='${dtype}',
+  little_endian=${LITTLE_ENDIAN},
+  rad_area='${RAD_AREA}',
+  netcdf=${netcdf_boolean},
+ /
+EOF
+
+	 startmsg
+         ./${angle_exec} < input >>   ${pgmout} 2>>errfile
+         export err=$?; err_chk
+         if [[ $err -ne 0 ]]; then
+             fail=$(expr $fail + 1)
+         fi
+
+#-------------------------------------------------------------------
```

Comment:
This is on me -- the comments from 195-209 are out of date and can be removed.

Reply:
Removed